paris-processor/src/main/java/com/airbnb/paris/processor/models/BaseStyleableInfo.kt

```diff
@@ -76,7 +76,8 @@ open internal class BaseStyleableInfo(
          */
         val elementType: TypeMirror,
         private val viewElementPackageName: String,
-        private val viewElementName: String,
+        /** The simple name of the view eg. "AirImageView" */
+        val viewElementName: String,
```

Comment:
I needed to access the name. Not sure if there was a reason it was private

Reply:
Nah it was just private 'cause it wasn't needed anywhere else. Totally fine to open it up.